src/Symfony/Component/Security/Core/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.2
+---
+
+* Deprecate the `Security` class, use `Symfony\Bundle\SecurityBundle\Security\Security` instead
```

Comment:
We should add the inverse side of this changelog entry to SecurityBundle's changelog